trino/trino_test.go

```diff
@@ -1070,6 +1206,42 @@ func TestTypeConversion(t *testing.T) {
 			ResponseUnmarshalledSample: "2017-07-10 01:02:03.000-04:00",
 			ExpectedGoValue:            time.Date(2017, 7, 10, 1, 2, 3, 0, time.FixedZone("", -4*3600)),
 		},
+		{
+			DataType:                   "timestamp",
+			RawType:                    "timestamp",
+			ResponseUnmarshalledSample: "2017-07-10 01:02:03.123456789",
+			ExpectedGoValue:            time.Date(2017, 7, 10, 1, 2, 3, 123456789, time.Local),
+		},
+		{
+			DataType:                   "timestamp with time zone",
+			RawType:                    "timestamp with time zone",
+			ResponseUnmarshalledSample: "2017-07-10 01:02:03.123456789 UTC",
+			ExpectedGoValue:            time.Date(2017, 7, 10, 1, 2, 3, 123456789, utc),
+		},
+		{
+			DataType:                   "timestamp with time zone",
+			RawType:                    "timestamp with time zone",
+			ResponseUnmarshalledSample: "2017-07-10 01:02:03.123456789 +03:00",
```

Comment:
Can the Trino server send a geographical TZ, like `Europe/Warsaw`, not just an offset?

Reply:
Yes, I think so, I'll check this works.

Reply:
Added a couple of tests, this works.